docs/styles/typography.css

```
@@ -4,18 +4,69 @@
   src: url("../fonts/TheFutureMono01-Regular.otf");
 }
 
+@font-face {
+  font-family: "Inter";
+  src: url('https://rsms.me/inter/inter.css');
+}
+
+@font-face {
+  font-family: "Source Code Pro";
+  src: url('https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap');
+}
+
 pre>code {
   display: block;
   line-height: .95em;
 }
 
+h1, h2, h3, h4, h5 {
+  font-family: Inter, sans-serif;
```

Should we add Arial as a backup before the default `sans-serif`?

```suggestion
  font-family: Inter, arial, sans-serif;
```

---

So that we can ensure consistency if Inter doesn't load? I'd rather display the system font, rather than Arial.

Found this system stack we could use. `-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;`

---

Ok cool, seems great